sfaira/data/dataloaders/loaders/d10_1016_j_cell_2021_01_053/human_na_2021_na_ren_001.yaml

```diff
@@ -0,0 +1,52 @@
+dataset_structure:
+    dataset_index: 1
+    sample_fns:
+        - "COVID19_ALL.h5ad"
+dataset_wise:
+    author: "Ren, Xianwen"
+    default_embedding: "NA"
```

Comment:
Is there no UMAP or tsne in this object? If yes, you can just indicate the corresponding obsm key here, if this object does not have any just leave blank.

Reply:
The dataset has its embedding in `adata.obs.umap1` and `adata.obs.umap2` but I could put it to an obsm key in the load function. `create-dataloader`  fills `default_embedding` with  `"NA"` by default, maybe that should be changed then?

Reply:
Yes, @Zethson let s add this to #370